IdentityCore/src/parameters/MSIDBoundRefreshTokenRedemptionParameters.m

```diff
@@ -62,15 +64,21 @@ - (nonnull instancetype)initWithClientId:(nonnull NSString *)clientId
         _clientId = clientId;
         _scopes = scopes;
         _nonce = nonce;
+        _extraPayloadClaims = extraPayloadClaims;
+        _workplaceJoinInfo = workplaceJoinInfo;
     }
     return self;
 }
 
 - (nonnull NSMutableDictionary *)jsonDictionary
 {
     NSMutableDictionary *jsonDict = [NSMutableDictionary new];
+    for (NSString *key in self.extraPayloadClaims)
+    {
```

Comment:
Do we want to copy those values so that extraPayloadClaims won't be retained? `jsonDict[key] = [self.extraPayloadClaims[key] copy];`